File: kotlin/kotlin-add-items-to-list/example4.kt
// https://codevscolor.com/kotlin-add-items-to-list
fun main() {
    val givenList = mutableListOf("one", "two", "three")
    givenList.plusAssign("four")
    givenList.plusAssign("five")
    givenList.plusAssign(arrayOf("six", "seven", "eight"))

    println(givenList)
}

File: dart/iterate-map/example4.dart
// https://codevscolor.com/dart-iterate-map
main() {
  var numMap = Map();
  numMap["one"] = 1;
  numMap["two"] = 2;
  numMap["three"] = 3;
  numMap["four"] = 4;
  numMap["five"] = 5;

  for (var v in numMap.values) {
    print(v);
  }
}